### ISSUE TITLE
Cannot start manuskript due to import error

### ISSUE THREAD

**Issue by Weej1** [olivierkes/manuskript#179]
Hi all,

They say "When it rains, it pours!" guess that's the case with this issue! I was compiling the stuff requested in ticket #173 by @olivierkes when I decided it'd be a good idea to send over the `.msk` file I had worked on to create my document. I ran manukskript and was greeted with this:

```
Traceback (most recent call last):
  File "manuskript", line 11, in <module>
  File "/usr/local/lib/python3.5/dist-packages/PyInstaller/loader/pyimod03_importers.py", line 389, in load_module
  File "manuskript/main.py", line 8, in <module>
  File "/usr/local/lib/python3.5/dist-packages/PyInstaller/loader/pyimod03_importers.py", line 573, in load_module
ImportError: /usr/lib/x86_64-linux-gnu/mesa/libGL.so.1: undefined symbol: drmGetDevices2
Failed to execute script manuskript
```

I've tried 2 other builds by @gedakc but those still give me `Segmentation fault (core dumped)`. The above error is new to me. Im guessing it may have something to do with my recent updates to my OS. I'm using `elementaryOS 0.4.1 Loki`  which is built on top of `Ubuntu 16.04.03`

As always, any input is appreciated ! :)

**Comment by gedakc** [comment]
@Weej1 could you try using either manuskript-0.4.0-linux.zip (no spell check) or manuskript-0.4.0-1-linux64.zip (includes spell check) and delete the file `manuskript/libdrm.so.2`?

Does manuskript run for you after deleting the `libdrm.so.2` file?

**Comment by Weej1** [comment]
@gedakc That fixed it! Not only in the original `manuskript-0.4.0-linux.zip` download but also in all the other versions I had (i.e. `manuskript-040-1-deb9` and `manuskript-0.4.0-1-linux64` are BOTH working now - no more segmentation fault errors). Does that file do anything of use even? Thanks again, I guess I can close this one!

**Comment by gedakc** [comment]
Thank you @Weej1 for testing this suggesting and confirming that it worked for you.  A similar solution worked in [I115 - Unable to launch in Linux Mint 18.1 Cinnamon 64-bit](https://github.com/olivierkes/manuskript/issues/115#issuecomment-330064175). 

From a quick Internet search I learned the following:

> libdrm provides a user space library for accessing the DRM, direct rendering manager, on operating systems that support the ioctl interface. libdrm is a low-level library, typically used by graphics drivers such as the Mesa DRI drivers, the X drivers, libva and similar projects. [Reference: Linux from scratch](http://www.linuxfromscratch.org/blfs/view/cvs/x/libdrm.html) 

I think this library is specific to the graphics card installed on a computer.  As such each GNU/Linux distro should already have a libdrm library installed for the specific hardware on the compuer.

I will investigate to see if I can remove this library from all the GNU/Linux packages to prevent this issue in the future.

**Comment by olivierkes** [comment]
That's awesome !

I had a quick exchange with the creator of [Anki](https://apps.ankiweb.net/), he told me the biggest issue for him was to find which files to remove from package. He told me we could have a look at his packages to see what he left. (But he uses PyQt4 if I remember correclty)

**Comment by gedakc** [comment]
Thanks @olivierkes for checking with Anki.  I've looked at that project but was unable to find the code that creates the package and more specifically identifies which files to delete.  Further PyQt4 is indeed quite different from PyQt5 as I've discovered when migrating sample code.

I've also looked at Calibre and it's quite another approach to making installable files.

If someone with knowledge and ambition steps up to create proper packages then that would be great.  In the meantime I'll stick with our PyInstall packages and over time we'll learn how to improve.
